src/main/resources/db/migration/V1_12__prisoner_iep_nullable_reviewed_by.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE prisoner_iep_level ALTER COLUMN reviewed_by DROP NOT NULL;
```

Comment:
Named this `V1_12` because in https://github.com/ministryofjustice/hmpps-incentives-api/pull/161 we have `V1_11` and wanted to avoid naming conflicts as well as Flyway confusion

Reply:
Thanks :) (I'll probably merge that PR later this morning anyway 👍 )